libimagcounter/src/counter.rs

```
@@ -110,62 +111,57 @@ impl<'a> Counter<'a> {
         match header.read("counter.value") {
             Ok(Some(Value::Integer(i))) => {
                 header.set("counter.value", Value::Integer(i - 1))
-                    .map_err(|e| CE::new(CEK::StoreWriteError, Some(Box::new(e))))
+                    .map_err_into(CEK::StoreWriteError)
                     .map(|_| ())
             },
-            Err(e) => Err(CE::new(CEK::StoreReadError, Some(Box::new(e)))),
+            Err(_) => Err(CEK::StoreReadError.into_error()),
```

We are hiding an error here, please undo this change.